### PR TITLE
Add description into the build files

### PR DESCRIPTION
**Pull request by SviatlanaHlukhava**
Add description into the build files

### CODE REVIEW

File: dirty_build.bat [hunk 1]
```diff
@@ -1 +1,6 @@
+rem test set: jUnit tests for java code
```
Comment (RomanKonovalov) [inline dirty_build.bat]: Does this batch file invoke unit tests?

File: commit_build.bat [hunk 2]
```diff
@@ -1 +1,6 @@
+rem test set: jUnit tests for java and javascript, integration tests
```
Comment (UladzimirKapylou) [inline commit_build.bat]: I would remove the word "set" from the first line of each file. Let's leave only "tests: " and further info.

File: commit_build.bat [hunk 3]
```diff
@@ -1 +1,6 @@
...
+rem code quality checks: yes (findbugs, pmd, checkstyle)
+rem deploy: no deploy
+rem intent of using: production
+rem minification: yes
 mvn clean install -Pprod
```
Comment (UladzimirKapylou) [inline commit_build.bat]: I know, you just added comments, but why do we need an "install" goal here (and in other scripts as well)?

File: dirty_deploy.bat [hunk 4]
```diff
@@ -1 +1,6 @@
+rem test set: jUnit tests for java code
+rem code quality checks: no
+rem deploy: deploy on local server tomcat 7
+rem intent of using: development
```
Comment (UladzimirKapylou) [inline dirty_deploy.bat]: We have two identical intent. How I can (as a developer) decide which file is better to use? Please add some more description here (and possibly in other files)

File: commit_build.bat (hunk 3 above)
Comment (RomanKonovalov) [inline commit_build.bat]: I think we should try to use verify goal

File: dirty_build.bat [hunk 5]
```diff
@@ -1 +1,6 @@
...
+rem code quality checks: no
+rem deploy: no deploy
+rem intent of using: development
+rem minification: no
 mvn clean install -DskipTests -DskipITs -Dfindbugs.skip=true -Dpmd.skip=true -Dcheckstyle.skip=true
```
Comment (RomanKonovalov) [inline dirty_build.bat]: I think package goal is more suitable

File: commit_build.bat (hunk 3 above)
Comment (UladzimirKapylou) [inline commit_build.bat]: Why verify?

File: dirty_deploy.bat [hunk 6]
```diff
@@ -1 +1,6 @@
...
+rem minification: no
 mvn clean install tomcat7:deploy -DskipTests -DskipITs -Dfindbugs.skip=true -Dpmd.skip=true -Dcheckstyle.skip=true
```
Comment (RomanKonovalov) [inline dirty_deploy.bat]: I think package goal is more suitable

File: commit_build.bat (hunk 3 above)
Comment (RomanKonovalov) [inline commit_build.bat]: We should invoke integration-test goal which goes after package one

Comment (UladzimirKapylou) [inline commit_build.bat]: Ok. Why you don't want to use exactly that one you need?

Comment (RomanKonovalov) [inline commit_build.bat]: Let's try ...

**Comment by RomanKonovalov** [comment]
:+1:

**Comment by UladzimirKapylou** [comment]
:+1:

**Pull request merged**